praw/models/reddit/live.py

```diff
@@ -37,3 +59,96 @@ def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
 
     def _info_path(self):
         return API_PATH['liveabout'].format(id=self.id)
+
+    @property
+    def fullname(self):
+        """Return ``name`` of :class:`.LiveThread`."""
+        return 'LiveUpdateEvent_' + self.id
+
+    def updates(self, **generator_kwargs):
+        """Return a :class:`.ListingGenerator` yields :class:`.LiveUpdate` s.
+
+        :param generator_kwargs: keyword arguments passed to
+            :class:`.ListingGenerator` constructor.
+        :returns: A :class:`.ListingGenerator` object which yields
+            :class:`.LiveUpdate` object.
+        """
+        path = API_PATH['live_updates'].format(id=self.id)
+        for update in ListingGenerator(self._reddit, path,
+                                       **generator_kwargs):
+            update.thread_id = self.id
+            yield update
+
+
+class LiveUpdate(RedditBase):
+    """An individual :class:`.LiveUpdate` object."""
+
+    STR_FIELD = 'id'
+
+    def __init__(self, reddit, thread_id=None, update_id=None, _data=None):
+        """Initialize a lazy :class:`.LiveUpdate` instance.
+
+        Either ``thread_id`` and ``update_id``, or ``_data`` must be
+        provided.
+
+        .. warning:: In the current implementation, accessing lazy attributes
+           (e.g., ``update.author``) is very slow.
+
+        :param reddit: An instance of :class:`.Reddit`.
+        :param thread_id: A live thread ID, e.g., ``'ukaeu1ik4sw5'``.
+        :param update_id: A live update ID, e.g.,
+            ``'7827987a-c998-11e4-a0b9-22000b6a88d2'``.
+
+        .. code-block:: python
+
+           update = LiveUpdate(reddit, 'ukaeu1ik4sw5',
+                               '7827987a-c998-11e4-a0b9-22000b6a88d2')
+           update.thread_id  # 'ukaeu1ik4sw5'
+           update.id         # '7827987a-c998-11e4-a0b9-22000b6a88d2'
+           update.author     # 'wting'
+        """
+        if thread_id and update_id and not _data:
+            super(LiveUpdate, self).__init__(reddit, _data)
+            self.thread_id = thread_id
+            self.id = update_id  # pylint: disable=invalid-name
+        elif (not thread_id and not update_id) and _data:
```

Comment:
Could this just be `if _data is not None` and be used as the first conditional? If `_data` isn't structured properly it's okay for it to blow-up since it's a private keyword anyway.